core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -291,10 +292,16 @@ protected[core] object WhiskWebActionsApi extends Directives {
     }
 }
 
+protected[controller] trait RespondWithMethods extends RespondWithHeaders {
```

Comment:
I find this trait confusing - you can add the parent trait to the web actions api and override there.
I don't think we're gaining anything from this.